### DISCUSSION TITLE
Zoomer programmatically changeable

### DISCUSSION THREAD

**Discussion by thhart** [GSI-CS-CO/chart-fx#391] [Q&A] [answered]
I would like to be able to influence the zoom frame when the chart is filled with data. Is this somehow possible via the standard Zoomer plugin?

**Comment by wirew0rm** [comment, marked as answer]
As for your other question, the simplest way is to just change the data range of the axes (`xAxis.set(min, max)`). The zoomer will still work, but it will not be able to restore the old state by right clicking. You could add the current state manually to the deqeue returned by `zoomer.zoomStackDequeue` (look at the (private) `pushCurrentZoomWindows()` function to see what you would have to add there). On the other hand, panning and scroll-zoom are also not restoreable.
